nautobot/utilities/deprecation.py

```diff
@@ -5,7 +5,7 @@
 import traceback
 import warnings
 
-from django.conf import settings
+from nautobot.core.settings import LOG_DEPRECATION_WARNINGS
```

Comment:
We should probably move this out of nautobot.core.settings entirely and just check the environment variable directly instead.